app/models/github_repo.rb

```diff
@@ -27,9 +27,8 @@ def self.upsert(user, **params)
   end
 
   def self.update_to_latest
-    where("updated_at < ?", 1.day.ago).find_each do |repo|
-      user = User.find_by(id: repo.user_id)
-      next unless user
+    where("updated_at < ?", 26.hours.ago).includes(:user).find_each do |repo|
```

Comment:
eager load users and extend lookback to 26 hours just in case an outage occurs, we have a little overlap. Plus it wont run at the exact same time each day since we poll for cron jobs every 10 seconds so without the overlap there is a chance that a repo could fall in that small window and get missed.